src/chrome/content/rules/TI.com-falsemixed.xml

```diff
@@ -1,31 +0,0 @@
-<!--
-	For rules not causing false/broken MCB, see Texas-Instruments.xml.
-
--->
-<ruleset name="TI.com (false MCB)" platform="mixedcontent">
-
-	<!--	Direct rewrites:
-				-->
-	<target host="www.ti.com" />
-
-	<!--	Complications:
-				-->
-	<target host="ti.com" />
-
-		<!--	Breaks doc downloads:
-						-->
-		<exclusion pattern="^http://www.ti.com/(?:general/docs/)?lit/" />
-
-			<!--	+ve:
-					-->
-			<test url="http://www.ti.com/general/docs/lit/getliterature.tsp?baseLiteratureNumber=spruer9&amp;fileType=pdf" />
-			<test url="http://www.ti.com/lit/ug/spruer9d/spruer9d.pdf" />
-
-
-	<rule from="^http://ti\.com/"
-		to="https://www.ti.com/" />
```

Comment:
Please keep this rule and add `ti.com` as target (but keep the comment about the timeout).